.devcontainer/devcontainer.json

```diff
@@ -22,6 +22,6 @@
     "SEE_API_CLIENT_SECRET": {}
   },
   "features": {},
-  "postStartCommand": "while [ \"$(python manage.py showmigrations | grep \"\\[ \\]\" | wc -l)\" -ne \"0\" ]; do echo \"waiting for migrations\"; sleep 2; done && python manage.py update_offered_by && python manage.py update_platforms && python manage.py update_departments_schools && python manage.py backpopulate_resource_channels --overwrite --all && python manage.py update_course_number_departments && python manage.py backpopulate_mitxonline_data && python manage.py backpopulate_micromasters_data && python manage.py recreate_index --all",
+  "postStartCommand": "while [ \"$(python manage.py showmigrations | grep \"\\[ \\]\" | wc -l)\" -ne \"0\" ]; do echo \"waiting for migrations\"; sleep 2; done && python manage.py update_offered_by && python manage.py update_platforms && python manage.py update_departments_schools && python manage.py backpopulate_resource_channels --overwrite --all && python manage.py update_course_number_departments && python manage.py backpopulate_mitxonline_data && python manage.py backpopulate_micromasters_data && python manage.py backpopulate_resource_channels --overwrite --all && python manage.py recreate_index --all",
```

Comment:
Why do we do `python manage.py backpopulate_resource_channels --overwrite --all` twice?

Currently in my dev container, I am seeing many topic channels without resources. They should be unpublished.

IMO the best solution for this is probably to channel the model's default value of `published` from True to False. But I suspect it might also be fixed by removing the second instance `python manage.py backpopulate_resource_channels --overwrite --all`.

I think:
1. First, `python manage.py backpopulate_resource_channels --overwrite --all` runs and all the channels get created
2. The resource backpopulation runs, which triggers unpublishing of many channels (since most don't have resources associated, they should not be published).
3. `python manage.py backpopulate_resource_channels --overwrite --all` runs again, and the channels all get re-created from scratch, with published=True.
    - Again, I think the default should be `published=False`

Reply:
removed the duplicate call to backpopulate.  This only affects the codespace environment at the moment. If we need to address the published=False as a default I'm thinking it might be better to do that in a separate ticket